mongodb.cpp

```diff
@@ -185,6 +185,7 @@ static class MongoDBExtension : public Extension {
 			/* MongoDb\Driver\ReadConcern */
 			HHVM_MALIAS(MongoDB\\Driver\\ReadConcern, __construct, MongoDBDriverReadConcern, __construct);
 			HHVM_MALIAS(MongoDB\\Driver\\ReadConcern, __debugInfo, MongoDBDriverReadConcern, __debugInfo);
+			HHVM_MALIAS(MongoDB\\Driver\\ReadConcern, bsonSerialize, MongoDBDriverReadConcern, __debugInfo);
```

Comment:
I assume `__debugInfo` is intentional here, since you're using the same method?

If we end up supporting `jsonSerialize()` down the line, I assume that will use the same method as well?